Automattic/wp-calypso
# ExPlat: Add /lib log-error

#### Changes proposed in this Pull Request

* This PR adds log-error

~~I haven't finished testing it - unit tests, and testing the SSR side of things - but this is needed to PR the other internals.~~

Part of a series rebuilding the reference WIP PR #48475 (Although no longer kept in sync).


#### Testing instructions



* Added unit tests

**Manual Testing:**

Run `logError` on the server side under docker:
1. Set `process.env.CALYPSO_LOGFILE = '/tmp/calypso.log';`
2 Add a `logError` function call in `client/server/index`
3. Run the docker build
4. Confirm the message has been logged correctly. 

Run `logError` in the client:
1. Add  a `logError` function call somewhere in the codebase e.g. the boot code
2. Visit the page in your browser
3. Confirm it logs to the console in development mode
4. Search LogStash for the message just sent, filter by `properties.context = "explat"`
